Library/Formula/miller.rb

```diff
@@ -0,0 +1,25 @@
+class Miller < Formula
+  desc "Miller is like sed, awk, cut, join, and sort for name-indexed data such as CSV."
+  homepage "http://johnkerl.org/miller/doc/index.html"
+  url "https://github.com/johnkerl/miller/archive/v1.0.0.tar.gz"
+  sha256 "8e00d1e55609e2deb27537d385646ee4b3a4227070bdb1375ba7eff2a7e92708"
+
+  depends_on "flex"
+
+  def install
+    ENV.deparallelize
+    system "make"
+    # makefile hardcodes path to install the mlr executable as of 1.0.0.
+    # This is some creative wankery to get around that, in two steps:
+    # Step 1 - create the bin dir the makefile is expecting.
+    mkdir bin
+    # Step 2 - temporarily manipulate the value of $HOME to the cellar prefix homebrew provides.
+    system "HOME=\"#{prefix}\"; make install"
+  end
+
+  test do
+    # in lieu of any -v we could test a version from, lets just assert that there
```

Comment:
`tmux` has a fairly vanilla test because there's not really much you can do with `tmux` that doesn't involve launching it, and throwing a new window is problematic in terms of testing.

Things can (and have) passed a `make check` stage just fine and then broken after installation for various reasons. The `test do` block exists to make sure that doesn't happen, and if one of the dependencies is upgraded that this can still build and run as expected. Ideally, we would use a functionality test everywhere, but vanilla tests should only be used when there's very little other option.